maestro-orchestra-models/src/main/java/maestro/orchestra/Commands.kt

```diff
@@ -51,11 +51,15 @@ data class SwipeCommand(
     val elementSelector: ElementSelector? = null,
     val startRelative: String? = null,
     val endRelative: String? = null,
-    val duration: Long = DEFAULT_DURATION_IN_MILLIS
+    val duration: Long = DEFAULT_DURATION_IN_MILLIS,
+    val label: String? = null,
 ) : Command {
 
     override fun description(): String {
         return when {
+            label != null && label != "" -> {
```

Comment:
```suggestion
            label != null -> {
```

See my comment on `Condition.kt`